extension/src/openvic-extension/classes/GFXMaskedFlagTexture.cpp

```diff
@@ -33,23 +31,16 @@ Error GFXMaskedFlagTexture::_generate_combined_image() {
 	if (mask_image.is_valid() && flag_image.is_valid() && flag_image_rect.has_area()) {
 		const Vector2i centre_translation = (mask_image->get_size() - button_image->get_size()) / 2;
 
-		for (
-			Vector2i combined_image_point { 0, 0 };
-			combined_image_point.y < button_image->get_height();
-			++combined_image_point.y
-		) {
-
+		for (Vector2i combined_image_point { 0, 0 }; combined_image_point.y < button_image->get_height();
+			 ++combined_image_point.y) {
```

Comment:
Here and with the `if` statement below, I'm not a fan of single breaks in the middle of a pair of brackets, I find it much clearer to always have matching breaks after `(` and before `)` if any break is needed between the brackets. I believe therre is a setting that does this for `{}` curly brackets very well, but idk if there's a `()` equivalent.